src/content/features/add-match-room-player-links.js

```diff
@@ -70,6 +71,7 @@ export default async parent => {
         .filter(
           key => user.socials[key].value && user.socials[key].value !== ''
         )
+        .filter(key => validateSocialLink(key, user.socials[key].value))
```

Comment:
This could be more descriptive:
```suggestion
        .filter(platform => validateSocialLink(platform, user.socials[key].value))
```